packages/voter-dapp/src/ActiveRequests.js

```diff
@@ -735,7 +753,12 @@ function ActiveRequests({ votingAccount, votingGateway }) {
           })}
         </TableBody>
       </Table>
-      {revealButtonShown ? (
+      {snapshotButtonShown && (
+        <Button variant="contained" color="primary" onClick={() => onSnapshotHandler()}>
```

Comment:
It might be worth wrapping a <Tooltip> component around this button to explain succinctly what the snapshot does. https://material-ui.com/components/tooltips/